co2eq/purchase/footprints.yml

```diff
@@ -6446,7 +6455,7 @@ _children:
               AT: 0.1374337367650893
               AU: 0.16397221933734732
               BE: 0.37984171220587076
-              BG: 0.5739356484816667
+              BG: 0.57393564848166670
```

Comment:
@pierresegonne why was this changed? I cannot really see that from the commits.

Reply:
To verify that people do review the PR.
No seriously it's because for some reason it was displayed differently in my editor and I thought that it would be parsed as a string. I'll remove the 0 to remove the change